debian/debathena-pyhesiodfs.service

```diff
@@ -0,0 +1,20 @@
+# Author: Lizhou Sha <slz@mit.edu>
+# This file is part of the debathena-pyhesiodfs package.
+
+[Unit]
+Description=Hesiod automounter for Athena lockers
+Requires=openafs-client.service
```

Comment:
Do we want the strong `Requires=` here, or would `Wants=` suffice? Do we want to enforce that if `afsd` was stopped, `pyhesiodfs` should also be stopped?